README.md

```diff
@@ -7,171 +7,133 @@ These are the evaluation criteria we use for performance evaluation and leveling
 3. Collaboration & Communication
 4. Community & Citizenship
 
-Jump to competencies for a particular level:
-
-* [Software Engineering Intern](#software-engineering-intern)
-* [Software Engineer](#software-engineer)
-* [Software Engineer 2](#software-engineer-2)
-* [Senior Software Engineer](#senior-software-engineer)
-* [Staff Software Engineer](#staff-software-engineer)
-
-
-## Software Engineering Intern
+Within each of these categories, we have further broken down the list into competencies that are applicable at every level and competencies that are only applicable at particular Software Engineer levels. Thes are designed to be cumulative; skills that are expected at Software Engineer 2, for example, are also expected at Senior Software Engineer & beyond. 
 
 ### Software Engineering & Design
 
-- You write efficient and readable code, you ask for help evaluating pros and cons of various approaches, and your code is easy to maintain.
+#### All Levels
+- You write efficient and readable code.
+- You evaluate pros and cons of various approaches, asking for help when needed.
+- Your code is easy to maintain.
 - You are learning and applying our standards around design, testing, error handling, monitoring, alerting, and documentation.
 - You proactively maintain, debug, and fix problems in your code.
 - You seek out and learn industry best practices and standards.
-- You work with your manager and senior engineers to write technical proposals and solicit feedback from the team. You attend architecture review discussions.
+- You write high quality technical proposals and solicit feedback from the team. 
+- You attend architecture review discussions.
 
-### Execution & Results
+#### Software Engineer Intern
 
-- You effectively manage your time and know when to ask for help to unblock yourself. When blocked, you are resourceful and take initiative in finding other tasks to complete.
-- You ask for help breaking down large tasks into smaller tasks that are easier to complete.
-- You ask for help investigating existing components and services when approaching new projects and feature requests.
+- You ask for help in order to meet the above competencies.
```

Comment:
This is definitely a flattening of some nuance that existed before, where some of the Intern competencies were phrased in terms of asking for help, and others phrased differently. I'm not sure the nuance we are losing was intentional or good;—just noting that it's gone.